code/modules/events/shuttle_catastrophe.dm

```diff
@@ -33,6 +43,11 @@
 	new_shuttle = pick(valid_shuttle_templates)
 
 /datum/round_event/shuttle_catastrophe/start()
+	if(SSshuttle.shuttle_insurance)
+		var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names